Dockerfile

```diff
@@ -1,36 +1,35 @@
-FROM node:8.9-alpine
+FROM node:alpine
 LABEL Description="This image is used to start the hc-api-feathers" Vendor="Human-Connection gGmbH" Version="1.0" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
 
-# update unix packages
-RUN apk update && apk upgrade
-RUN rm -rf /var/cache/apk/*
-RUN yarn global add pm2
-
 # expose the app port
 EXPOSE 3030
 
-# set environment variables
-# ENV NPM_CONFIG_PRODUCTION=false
-# ENV HOST=0.0.0.0
-ENV NODE_ENV=production
-ENV API_PORT=3030
+# create working directory
+RUN mkdir -p /API
+WORKDIR /API
 
-# start the application in a autohealing cluster
-# NOTE: quick fix for server issues, restart api when reaching max of 300 MB Memory Usage (happens in conjunction with 100% CPU Usage)
-# TODO: find better way of dealing with that issue
-CMD NODE_ENV=production pm2 start server/index.js -n api --attach --max-memory-restart 1024M
-# CMD NODE_ENV=production pm2 start server/index.js -n api -i 2 --attach
-# as we have issues with pm2 currently in conjunction with nuxt, we use the standard approach here
-# CMD NODE_ENV=production node server/index.js
+# --no-cache: download package index on-the-fly, no need to cleanup afterwards
+# --virtual: bundle packages, remove whole bundle at once, when done
+RUN apk --no-cache --virtual build-dependencies add python make g++
 
-# create working directory
-RUN mkdir -p /var/www/
-WORKDIR /var/www/
+RUN yarn global add pm2
 
 # install app dependencies
-COPY package.json /var/www/
-COPY yarn.lock /var/www/
+COPY package.json /API
+COPY yarn.lock /API
 RUN yarn install --frozen-lockfile --non-interactive
 
```

Comment:
Nachdem die Abhängigkeiten installiert wurden musste ich noch `RUN yarn global add cross-env nodemon` ausführen, weil das starten der API und mongod sonst nicht ging. Gerade startet bei mir nur die API nicht. Es kann `server-favicon` nicht geladen werden..

Reply:
I removed all my images, volumes and containers from the cache and rebuild everything and I cannot reproduce that error. Super weird.